packages/at_onboarding_cli/lib/src/cli/auth_cli.dart

```diff
@@ -32,10 +48,14 @@ Future<int> main(List<String> arguments) async {
     stderr.writeln('Error: $e');
     aca.parser.printAllCommandsUsage();
     return 1;
+  } finally {
+    try {
+      deleteStorage();
+    } catch (_) {}
   }
 }
 
-Future<int> _main(List<String> arguments) async {
+Future<int> wrappedMain(List<String> arguments) async {
```

Comment:
made public so that it can be called by the at_onboarding_cli_test in at_onboarding_cli_functional_tests